ci.sh

```diff
@@ -135,8 +135,23 @@ sleep 5
 pgrep -f target/debug/parsec >/dev/null
 
 if [ "$PROVIDER_NAME" = "all" ]; then
-    echo "Execute all-providers tests"
+    echo "Execute all-providers normal tests"
     RUST_BACKTRACE=1 cargo test $TEST_FEATURES --manifest-path ./e2e_tests/Cargo.toml all_providers::normal
+
+    echo "Execute all-providers multi-tenancy tests"
+    su -c "RUST_BACKTRACE=1 cargo test $TEST_FEATURES --manifest-path ./e2e_tests/Cargo.toml --target-dir /home/parsec-client-1 all_providers::multitenancy::client1_before" parsec-client-1
```

Comment:
I was being a bit too dramatic. The problem was that each user was trying to write the `target` directory owned by `root`. Using the `--target-dir` option fixed the problem 🚀